src/Application/UseCases/Product/Register/ProductResponse.php

```diff
@@ -4,10 +4,10 @@
 
 namespace App\Application\UseCases\Product\Register;
 
-use App\Application\UseCases\RegisterResponse;
+use App\Application\UseCases\Response;
 use App\Domain\Model\Product\Product;
 
-final class RegisterProductResponse extends RegisterResponse
```

Comment:
Bad naming. ProductResponse class=> RegisterProductResponse class.

Reply:
This changes will submitted for the next feature.